pypit/tests/files/LRISb_600_WaveCalib_01.json

```diff
@@ -0,0 +1,2139 @@
+{
```

Comment:
Wait until I start uploading Mb files! ;)
But actually, I need the spectrum that is in here to run the test so
fear this one needs to be a bit larger.